hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -701,20 +701,17 @@ tests:
   # Simulating metric resets at time slots 5 and 9
   - interval: 1h
     input_series:
-      - series: 'kubevirt_migrate_vmi_succeeded_total{vmi="vmi-example-1", source="node-1", target="node-2"}'
-        # time:  0 1 2 3 4 5 6 7 8 9 10 11 12 13
-        values: "_ 1 2 2 3 _ 1 2 2 _ _  1  2  3+0x13"        # 7 increases, in samples: 2,4,6,7,11,12,13
-      - series: 'kubevirt_migrate_vmi_succeeded_total{vmi="vmi-example-1", source="node-2", target="node-1"}'
-        # time:  0 1 2 3 4 5 6 7 8 9 10 11 12 13
-        values: "_ _ 1 2 2 _ _ 1 2 _ 1  2  2  2+0x13"        # 6 increases, in samples: 2,3,7,8,10,11
+      - series: 'kubevirt_migrate_vmi_succeeded_total{vmi="vmi-example-1", vmim="vmi-example-1-migration"}'
+        # time:  0 1 2 3 4 5
+        values: "_ _ _ _ _ 13"
 
     alert_rule_test:
-      # at 11h, there are total of 11 migrations made on a single VMI, so the alert should not be fired.
-      - eval_time: 11h
+      # at 4h, there are total of 11 migrations made on a single VMI, so the alert should not be fired.
+      - eval_time: 4h
```

Comment:
Why did the evaluation time changed?
Perhaps it deserve a separate commit with an explanation, because this is actually a functionality change rather than a refactoring

Reply:
After the change explained above, I just moved the times back. I think it makes no difference, everything just happens sooner.

I'll move the alert changes into a separate commit.